app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -139,7 +139,12 @@
 
       <% summary_list.row(html_attributes: { data: { qa: "course__start_date" } }) do |row| %>
         <% row.key { "Course starts" } %>
-        <% row.value { l(course.start_date&.to_date, format: :short) } %>
+        <% row.value do %>
+          <ul class="govuk-list">
+            <li><%= l(course.start_date&.to_date, format: :short) %></li>
+            <li>Academic year <%= course.cycle_range %></li>
+          </ul>
```

Comment:
Semantically this is wrong as it not a list

Reply:
You could think of it as a list of Course start details, if you do something like 
```
<%= l(course.start_date&.to_date, format: :short) %><br>
            Academic year <%= course.cycle_range %>
```
 it does not match the spacing in the locations list higher up the page. Am happy to take suggestions...

Reply:
```html
          <p class="govuk-body">September 2022</p>
          <p class="govuk-hint">Academic year 2022 to 2023</p>
```

Something closer to prototype...

Reply:
See above